doc/services/portability/posix/overview/index.rst

```diff
@@ -0,0 +1,131 @@
+
+.. _posix_subprofiles:
+
+POSIX Subprofiles
+=================
+
+While Zephyr supports running multiple `threads <threads_v2>`_ (possibly in an `SMP <smp_arch>`_
+configuration), as well as `Virtual Memory and MMUs <x86_developer_guide>`_, Zephyr code and data
+normally share a common address space. The Zephyr kernel executable code and the application
+executable code are typically compiled into the same binary artifact. From that perspective, Zephyr
+apps can be seen as running in the context of a single process.
+
+While multi-purpose operating systems (OS) offer full POSIX conformance, Real-Time Operating
+Systems (RTOS) such as Zephyr typically serve a fixed-purpose, have limited hardware resources,
+and experience limited user interaction. In such systems, full POSIX conformance can be
+impractical and unnecessary.
+
+For that reason, POSIX defined the following :ref:`Application Environment Profiles (AEP)<posix_aep>`
+as part of `IEEE 1003.13-2003`_ (also known as POSIX.13-2003).
+
+* Minimal Realtime System Profile (:ref:`PSE51 <posix_aep_pse51>`)
+* Realtime Controller System Profile (:ref:`PSE52 <posix_aep_pse52>`)
+* Dedicated Realtime System Profile (:ref:`PSE53 <posix_aep_pse53>`)
+* Multi-Purpose Realtime System (PSE54)
+
+POSIX.13-2003 AEP were formalized in 2003 via "Units of Functionality" but the specification is now
+inactive (for reference only). Nevertheless, the intent is still captured as part of POSIX-1.2017
+via :ref:`Options<posix_options>` and :ref:`Option Groups<posix_option_groups>`, in Appendix E.
+
+.. _posix_apps:
+
+POSIX Applications in Zephyr
+============================
+
+A POSIX app in Zephyr is :ref:`built like any other app<application>`; it requires the usual
+`prj.conf` and `CMakeLists.txt`, and then it is :ref:`built the usual way<samples-and-demos>`.
+
+.. code-block:: kconfig
+   :caption: `prj.conf` for a simple POSIX app in Zephyr
+
+    CONFIG_POSIX_API=y
+
+.. code-block:: c
+   :caption: A simple app that uses Zephyr's POSIX API
+
+    #include <stddef.h>
+    #include <stdio.h>
+    #include <time.h>
+
+    void megasleep(size_t megaseconds)
+    {
+        struct timespec ts = {
+            .tv_sec = megaseconds * 1000000,
+            .tv_nsec = 0,
+        };
+
+        printf("See you in a while!\n");
+        if (nanosleep(&ts, NULL) == -1) {
+            perror("nanosleep");
+        }
+    }
+
+    int main()
+    {
+        megasleep(42);
+        return 0;
+    }
+
+..
+    TODO: insert a link to a list of all samples tagged with 'posix'
+
+.. _posix_config:
+
+Configuration
+=============
+
+Like many features in Zephyr, POSIX features are configurable but
+:ref:`disabled by default<introducing_zephyr>`. Users must explicitly choose to enable POSIX
+options via :ref:`Kconfig<kconfig>` selection.
+
+Alternatively, users may enable one of the Kconfig options below as a shortcut to enable multiple
+:ref:`Option Groups<posix_option_groups>`.
+
+* :kconfig:option:`CONFIG_POSIX_API`
```

Comment:
It would be good to have a page that lists all possible POSIX Kconfig options (including those that are currently in net or whatnot).

Ideally that could be semi-automated, but I think that's a future problem.

Similarly, it would be good to automatically generate a doxygen stub for functions.

For this PR, I could list them statically.

Reply:
Fixed! (see Configuration Options)